tests/bluetooth/hci/prj.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_TEST=y
+CONFIG_ZTEST=y
+
+CONFIG_BT=y
+CONFIG_BT_CTLR=n
+CONFIG_BT_NO_DRIVER=y
```

Comment:
Just a note that this last addition actually conflicts with #72323 since that removes the `BT_NO_DRIVER` options completely. So we'll need to pay attention to which PR gets merged first.

Reply:
Yes, I noticed :)

Reply:
#72323 was merged, so you'll need to remove the `BT_NO_DRIVER` reference now. I'm not sure it's necessary, but for some other unit tests I replaced it with `CONFIG_BT_H4=n`, since that may get enabled on some target that these tests are built against.

Reply:
Done